src/v3/src/mocks/scenario/securityquestion-enroll-mfa.ts

```diff
@@ -61,13 +61,15 @@ scenario('securityquestion-enroll-mfa', (rest) => ([
     const request = req.body as Record<string, any>;
     const answer = request.credentials?.answer;
     let response = null;
+    let responseStatus = 200;
     if (answer?.length < 4) {
       response = (await import('../response/idp/idx/challenge/answer/enroll-security-question-with-character-limit-error.json')).default;
+      responseStatus = 400;
     } else {
       response = (await import('../response/oauth2/default/v1/token/default.json')).default;
     }
     return res(
-      ctx.status(200),
+      ctx.status(responseStatus),
       ctx.json(response),
     );
```

Comment:
suggest that we move the `return`s to inside the `if/else` so the `responseStatus` variable is no longer needed.